.gitignore

```diff
@@ -27,4 +27,5 @@ hs_err_pid*
 ProductsFromMarketPlace.iml
 
 /.idea/
-/target/
\ No newline at end of file
+/target/
+/Plan.txt
```

Comment:
try adding the .iml files into gitignore and then use git rm to remove them from repo